gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineFindBugs.groovy

```diff
@@ -30,6 +30,7 @@ class BaselineFindBugs extends AbstractBaselinePlugin {
 
     private static final String DEFAULT_FINDBUGS_VERSION = '3.0.1'
     private static final String DEFAULT_EFFORT = 'max'
+    private static final String ANTIPATTERNS_ARTIFACT = 'com.palantir.antipatterns:antipatterns:1.0-beta.4'
```

Comment:
Can't you just override your antipatterns version by just setting

    findbugsPlugins 'com.palantir.antipatterns:1.1.0'

or whatever?

Won't that override it?